vehicle/autoware_vehicle_info_utils/src/vehicle_info.cpp

```diff
@@ -85,4 +85,16 @@ VehicleInfo createVehicleInfo(
   };
 }
 
+double VehicleInfo::calcMaxCurvature() const
+{
+  const double radius = wheel_base_m / std::tan(max_steer_angle_rad);
+  const double curvature = 1.0 / radius;
+  return curvature;
+}
+double VehicleInfo::calcCurvatureFromSteerAngle(const double steer_angle) const
+{
+  const double radius = wheel_base_m / std::tan(steer_angle);
+  const double curvature = 1.0 / radius;
+  return curvature;
+}
```

Comment:
I think it is better to consider 0 division.

Reply:
Thank you. Fixed. https://github.com/autowarefoundation/autoware.universe/pull/7432/commits/3d1743caa4d016688902051271feef90d1755581